scriptmodules/supplementary/skyscraper.sh

```diff
@@ -0,0 +1,696 @@
+#!/usr/bin/env bash -x
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="skyscraper"
+rp_module_desc="Scraper for EmulationStation by Lars Muldjord"
+rp_module_licence="GPLv3.0 https://raw.githubusercontent.com/muldjord/skyscraper/master/LICENSE"
+rp_module_section="opt"
+
+function depends_skyscraper() {
+    getDepends qt5-default p7zip-full
+}
+
+function sources_skyscraper() {
+    local latest_ver
+    latest_ver=$(_latest_ver_skyscraper)
+
```

Comment:
not sure there is any need for a variable here at all.

Reply:
OK, I removed it and put the call inline